### ISSUE TITLE
Problem with funcnet.coupling_analysis information_transfer

### ISSUE THREAD

**Issue by samanthamolnar** [pik-copan/pyunicorn#80]
Hello,

While trying to run the information transfer function on the test_data() provided in the coupling analysis class I run into a segmentation fault.  I think I have narrowed the problem down to the static method _get_nearest_neighbors.  I think the problem is with the C code that is being run with weave_inline.  Is anyone else experiencing this problem?

Here is my code:
```
from pyunicorn import funcnet
import numpy as np

data=funcnet.coupling_analysis.CouplingAnalysis.test_data()
coup_ana=funcnet.coupling_analysis.CouplingAnalysis(data)
mi=coup_ana.information_transfer(lag_mode='max')
```

-----------------------------------------------------------------
And here is the error:

> engr2-11-64-dhcp:Data sammolnar$ python2 test_unicorn.py 
> Segmentation fault: 11

**Comment by jakobrunge** [comment]
Dear Samantha,

it probably has to do with your C-compiler. Try the attached simply weave script.
[test_weave.py.tar.gz](https://github.com/pik-copan/pyunicorn/files/175982/test_weave.py.tar.gz)

 If that script doesn't work, you have to check your C-compilers... It depends on you operating system. I use the anaconda python environment in linux where scipy with weave is easiliy integrated. But anaconda also works on other platforms. Maybe also here you can find help with weave: http://scipy.github.io/old-wiki/pages/Weave

Hope this helps!
~Jakob

**Comment by samanthamolnar** [comment]
Hi Jakob,

I tried the weave script you provided and it printed out warnings but no
errors.  I am using anaconda as my environment.  For a while I was having
trouble with weave because they are no longer being supported by scipy but
when I fixed that problem then I started getting the segmentation faults.
I will try posting my question to weave and see what they say.  Thanks for
getting back to me.

-Sam




-Sam Molnar
